docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -123,3 +123,87 @@ Use *\\* for escaping of special characters in regular expressions
 * To consider failed assertion known or potentially-known *assertionPattern* should match assertion failure, if defined *variablePatterns* should match also.
 * If *assertionPattern*/*variablePatterns* matched and you have *story/scenario/step/Pattern* defined, to consider issue known all of them should match.
 * If *assertionPattern*/*variablePatterns* matched and you have *story/scenario/step/Pattern* defined, when at least one not matched issue is considered potentially known.
+
+== Meta filters
+
+Meta tags functionality can be used to reach several goals:
+
+* Scenarios' grouping in Allure report.
+* Scenarios' delimitation for test executing by selected tag.
+* Setting desired execution attributes on scenario-level.
+
+=== Levels
+Meta tags may be used at both story and scenario level. Meta tags specified at Story level are propagated to every scenario, if the same meta tag is used at Scenario level, then Scenario level value is applied.
+----
+Meta:
+     @group Login
+
+Scenario: Check redirection to Online Stores
+Meta:
+  @testTier 1
+  @layout desktop
+
+Scenario: Check redirection to Online Stores
+Meta:
+  @testTier 2
+  @layout desktop
+
+Scenario: Check redirection to Online Stores
+Meta:
+  @testTier 2
+  @layout desktop
+  @skip
+----
+
+Property *bdd.meta-filters* with groovy can be used to select scenarios for execution.
+Example:
+----
+bdd.meta-filters=groovy: (group == 'Login' && != skip)
+----
+selects all stories/scenarios with tag @group Login and without @skip.
+
+Generic meta tags can be used in test scenarios to markup scenarios appropriately following common rules:
+
+
+=== 1D tags (@key)
+[cols="1,4", options="header"]
+|===
+|Tag
+|Description
+
+|[subs=+quotes]`@proxy`
```

Comment:
this meta tag shouldn't be described here, since
- it's common documentation, but `@proxy` is a meta tag from plugin-web-app
- it's already described here: https://docs.vividus.dev/vividus/latest/plugins/plugin-web-app.html#_meta_tags